planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -655,18 +663,25 @@ void GoalPlannerModule::setLanes()
 void GoalPlannerModule::setOutput(BehaviorModuleOutput & output)
 {
   if (status_.is_safe_static_objects) {
-    // clear stop pose when the path is safe and activated
-    if (isActivated()) {
-      resetWallPoses();
-    }
+    if (isSafePath()) {
+      // clear stop pose when the path is safe against static/dynamic objects and activated
+      if (isActivated()) {
+        resetWallPoses();
+      }
+      // keep stop if not enough time passed,
+      // because it takes time for the trajectory to be reflected
+      auto current_path = getCurrentPath();
+      keepStoppedWithCurrentPath(current_path);
 
-    // keep stop if not enough time passed,
-    // because it takes time for the trajectory to be reflected
-    auto current_path = getCurrentPath();
-    keepStoppedWithCurrentPath(current_path);
+      output.path = std::make_shared<PathWithLaneId>(current_path);
+      output.reference_path = getPreviousModuleOutput().reference_path;
+    } else {
```

Comment:
maybe better for computational cost and readability
```suggestion
    } else if (status_.has_decided_path && isActivated()) {
```

Reply:
changed in https://github.com/autowarefoundation/autoware.universe/pull/4927/commits/22e940b8fe63cb712f6544d3130aab2a7dc799f9